openmdao/recorders/tests/test_sqlite_recorder.py

```diff
@@ -1298,7 +1298,7 @@ def test_record_solver_linear_block_gs(self):
             'pz.z': [0., 0.],
             'd1.y1': [-9.10908321e-11],
             'd2.y2': [0.],
-            'obj_cmp.obj': [2.03287907e-20],
+            'obj_cmp.obj': [2.28698896e-20],
```

Comment:
I guess the number changed a bit?

Reply:
Yeah, but I wasn't too worried about it since the number is so small.  Do you want me to spend the time to dig into it?

Reply:
no. I hadn't noticed that it was 1e-20 at first. Thats totally fine.